### ISSUE TITLE
V2.0.10 for Linux Application starting itself as Root User completely breaks Screen Reader Accessibility

### ISSUE THREAD

**Issue by the-swissionary** [raspberrypi/rpi-imager#1671]
### What happened?

Hello. I am completely blind an use the Orca Screen Reader for my daily work on my RaspberryPi 5; it runs a gnome Desktop, should this be relvant.
Sadly, I find that since a few months, I am no longer able to use RaspberryPi Imager on LInux at all. When I start the application, it prompts me for my Root password. After that, all I (or my Screen Reader) can see is the window Title "Raspberry Pi Imager 2.10.0 - Administrative Privileges", and nothing else. All the selection lists are simply no longer visible for my screen reader and thus for me.
As a workaround, I can skipt the Rood Password Prompt. then, the application starts, and the menus are amazingly accessible (Great Job!); however, the application now tells me it is missing its administrative privileges to actually write an image. So all the nice accessibility does not help at all.
Could you maybe check into this? And see if somehow, accessibility could be restore? (I remember that about a year ago, I was able to actually use the Imager on Linux with Orca; it was not the greatest experience, but it worked; so whatever happened in the meantime seems to cause the problems). Thanks for looking into this.

### Version

2.0.10 (Default)

### What host operating system were you using?

Debian and derivatives (eg Ubuntu)

### Host OS Version

Debian Trixie

### Selected OS

Didn't get the chance to select anything.

### Which Raspberry Pi Device are you using?

Raspberry Pi 5, 500, and Compute Modules 5

### What kind of storage device are you using?

Other

### OS Customisation

- [ ] Yes, I was using OS Customisation when the bug occurred.

### Relevant log output

```shell

```

**Comment by tdewey-rpi** [comment]
I'm investigating this issue - however, it's architectural in nature, which means it will likely target Imager 2.1
